pkg/config/agent/config.go

```diff
@@ -307,3 +309,23 @@ type PolicyBypassRule struct {
 	// The destination port of the given protocol.
 	Port int `yaml:"port,omitempty"`
 }
+
+type SecondaryNetworkOVSConfig struct {
```

Comment:
name is wrong

Reply:
Right. This one should be `SecondaryNetworkConfig`.